### DISCUSSION TITLE
Crud list buttons

### DISCUSSION THREAD

**Discussion by ale1981** [Laravel-Backpack/community-forum#720] [Q&A (Help)]
I have a widget which shows a models one to many relation as a table in the same way the list operation works. The issue I have at the moment is that I have to hard code the buttons available on the list rather than being able to create them dynamically depending on the operations that are available for the relation.

Is it possible for me to get the active operations as buttons for the relation from within the parent crud?

**Comment by karandatwani92** [comment]
Hey @ale1981 Sorry for taking so long to reply.

Currently, Backpack doesn't have an API for this. But as a workaround, you can try:
https://github.com/izica/relations-widgets-for-backpack

Let us know if this helped you out. Thanks
